cli/transaction.js

```diff
@@ -3,8 +3,10 @@ import { Command } from 'commander'
 const transactionController = new Command('transaction')
 
 transactionController
-    .command('log <from> <to> <amount>')
-    .description('Log transaction data to the console')
-    .action((from, to, amount) => {
-        // TODO
-    })
+  .command('log <from> <to> <amount>')
+  .description('Log transaction data to the console')
+  .action((from, to, amount) => {
+    // TODO
+  })
+
+export default transactionController
```

Comment:
```suggestion
```

This might've been left in from implementing the answer!

Reply:
> I couldn't think of a way of avoiding `import { Command }` from throwing an error so tried to sidestep it altogether and made them do an export from `exercises/hello.js`
> 
> Since I'd already made them export some stuff I left that one in, but they still have to import it into `cli/index.js`.

Ahhh, that makes sense! Learners encountering errors are a bit more risky when it comes to demanding trainer time as well. I think I'd want them to start encountering them more and more, and be able to read and understand them - but that'll likely need a trainer during the workshop for reassurance and guidance.